docs/EMERGENCY.md

```diff
@@ -0,0 +1,213 @@
+# Emergency Procedures for Yearn Finance
+
+## TLDR: [Emergency Check List](#emergency-checklist)
+
+## Introduction
+
+This document details the emergency procedures and guidelines that should take place during an emergency situation.
+
+The main goal of this document is to help minimize as much as possible a potential loss of funds in yearn platform and Smart Contracts during an emergency situation.
+
+## Definitions and Examples of Emergencies
+
+For the purposes of this document we define emergency situation to be:
+
+**_Any situation that may lead to a considerable amount of loss of funds for yearn users._**
+
+This is a non exhaustive list of possible emergency scenarios:
```

Comment:
Would add:

- Potential exploit discovered by team or bounty program researcher
- Active exploit discovered by unknown party and hack in progress